23.11 fb surgical rounds observation

Ohsudev opened the pull request:
Modified for Surgical Rounds input form to include new Clinical Observation input columns.

Ohsudev committed:
Modified Surgical Rounds form to add new program enhancements for Mike Ruez.

Ohsudev committed:
Merge branch 'release23.11-SNAPSHOT' into 23.11_fb_SurgicalRoundsObservation

Ohsudev committed:
Modified Surgical rounds observation columns.

Ohsudev committed:
Modified Surgical rounds observation columns.

Ohsudev committed:
Modified Surgical rounds observation columns.

Ohsudev committed:
Modified Surgical rounds observation columns.

Ohsudev committed:
Modified Surgical rounds observation columns.

Ohsudev committed:
Merge branch 'release23.11-SNAPSHOT' into 23.11_fb_SurgicalRoundsObservation

Ohsudev committed:
Modified Surgical Rounds input form to include new Clinical Observation input columns.

Ohsudev committed:
Modified for Surgical Rounds input form to include new Clinical Observation input columns.

Ohsudev committed:
Modified for Surgical Rounds input form to include new Clinical Observation input columns.

Ohsudev committed:
Modified for Surgical Rounds input form to include new Clinical Observation input columns.

Ohsudev committed:
Merge branch 'release23.11-SNAPSHOT' into 23.11_fb_SurgicalRoundsObservation

Ohsudev committed:
Modified Surgical Rounds form to include modified Clinical Observation new reportin columns.

Ohsudev committed:
Modified Surgical Rounds form to include modified Clinical Observation new reportin columns.

Ohsudev committed:
Modified Surgical Rounds form to include modified Clinical Observation new reportin columns.

Ohsudev committed:
Modified Surgical Rounds form to include modified Clinical Observation new reportin columns.

Ohsudev committed:
Modified Surgical Rounds form to include modified Clinical Observation new reportin columns.

Ohsudev committed:
Modified Surgical input forms.

Ohsudev committed:
Modified Surgical input forms.

Ohsudev committed:
Modified Surgical input forms.

Ohsudev committed:
Modified Surgical input forms.

Ohsudev committed:
Modified Surgical input forms.

Ohsudev committed:
Modified Surgical input forms.

Ohsudev committed:
Modified Surgical input forms.

Ohsudev committed:
Modified Surgical input forms.

Ohsudev committed:
Merge branch 'release23.11-SNAPSHOT' into 23.11_fb_SurgicalRoundsObservation

Ohsudev committed:
Modified Surgical input forms.

Ohsudev committed:
Merge branch 'release23.11-SNAPSHOT' into 23.11_fb_SurgicalRoundsObservation

Ohsudev committed:
Modified Surgical rounds project

Ohsudev committed:
Modified Surgical rounds project

Ohsudev committed:
Modified Surgical rounds project

Ohsudev committed:
Modified Surgery Rounds and Surgery input forms to include default values that needs to populate Cliniccal Observation entries.

Ohsudev committed:
Modified Surgery Rounds and Surgery input forms to include default values that needs to populate Cliniccal Observation entries.

Ohsudev committed:
Merge branch 'release23.11-SNAPSHOT' into 23.11_fb_SurgicalRoundsObservation

Ohsudev committed:
Modified Surgery input forms to include Suergery Procedure unique identifier.

Ohsudev committed:
Modified Surgery input forms to include Suergery Procedure unique identifier.

Ohsudev committed:
Modified Surgical Rounds Clinical Observation width display size

Ohsudev committed:
Modified Surgical Rounds Clinical Observation width display size

Ohsudev committed:
Modified Surgical Rounds Clinical Observation width display size

Ohsudev committed:
Modified Surgical Rounds Clinical Observation width display size

Ohsudev committed:
Modified Surgical Rounds Clinical Observation width display size

Ohsudev committed:
Modified Surgical Rounds Clinical Observation width display size

Ohsudev committed:
Modified Surgical Rounds Clinical Observation width display size

Ohsudev committed:
Modified Surgical Rounds Clinical Observation width display size

Ohsudev committed:
Merge remote-tracking branch 'origin/23.11_fb_SurgicalRoundsObservation' into 23.11_fb_SurgicalRoundsObservation

Ohsudev committed:
Merge branch 'release23.11-SNAPSHOT' into 23.11_fb_SurgicalRoundsObservation

Ohsudev committed:
Modified new Surgical Rounds program process to include default remarks values.

Ohsudev committed:
Merge branch 'release23.11-SNAPSHOT' into 23.11_fb_SurgicalRoundsObservation

Ohsudev committed:
Modified new Surgical Rounds program process to include default remarks values.

Ohsudev committed:
Modified new Surgical Rounds program process to include default remarks values.

Ohsudev committed:
Modified new Surgical Rounds program process to include default remarks values.

Ohsudev committed:
Modified new Surgical Rounds program process to include default remarks values.

Ohsudev committed:
Modified new Surgical Rounds program process to include default remarks values.

Ohsudev committed:
Modified Surgical Rounds program to exclude type of procedure values.

Ohsudev committed:
Modified Surgical Rounds validation process.

Ohsudev committed:
Modified Surgical Rounds validation process.

Ohsudev committed:
Modified Surgical Rounds validation process.

Ohsudev committed:
Modified Surgical Rounds validation process.

Ohsudev committed:
Modified Mense input form to allow error messages to be displayed when trigger scripts generates a warning.

Ohsudev committed:
Modified Surgical Rounds look-up tables.

Ohsudev committed:
Modified Surgical Rounds look-up tables.

Ohsudev committed:
Modified Surgical rounds case scripts.

Ohsudev committed:
Modified Surgical rounds case scripts.

Ohsudev committed:
Modified Surgical rounds case scripts.

Ohsudev committed:
Merge branch 'release23.11-SNAPSHOT' into 23.11_fb_SurgicalRoundsObservation

# Conflicts:
#	onprc_ehr/resources/schemas/dbscripts/sqlserver/onprc_ehr-23.015-23.016.sql

Ohsudev committed:
Modified Surgical rounds case scripts.

Ohsudev committed:
Modified Surgical Rounds process.

labkey-martyp committed:
Handle multiselect value

labkey-martyp commented:
@Ohsudev I believe this commit fixes the issue with the multi-select. Please check it out when you get a chance.

Ohsudev committed:
Modified Surgical Rounds was updated with additional code to allow hightlighting of multislect combo box selections.

Ohsudev committed:
Merge remote-tracking branch 'origin/23.11_fb_SurgicalRoundsObservation' into 23.11_fb_SurgicalRoundsObservation

# Conflicts:
#	onprc_ehr/resources/web/onprc_ehr/grid/ObservationsRowEditorGridPanel.js

Ohsudev committed:
Merge branch 'release23.11-SNAPSHOT' into 23.11_fb_SurgicalRoundsObservation

labkey-martyp reviewed (commented):
Let me know when you're ready to merge this in and I'll merge in the associated EHR PR.

Please add the new dataset columns to datasets_metadata.xml

Ohsudev commented:
Most of the code that I have added, and the code you modified should all have been added to
the ONPRC_EHR side of things.

You shouldn’t have to merge any code on the EHR side of things.

If I failed to convert a file to the ONPRC_EHR version, please let me know as soon as possible.

Always appreciated your help past, and present.

In addition, I have added the new set of columns to the file “datasets_metadata.xml” as you had instructed.

Thanks,
Raymond

From: Marty Pradere ***@***.***>
Sent: Wednesday, December 18, 2024 11:21 AM
To: LabKey/onprcEHRModules ***@***.***>
Cc: Raymond Blasa ***@***.***>; Mention ***@***.***>
Subject: [EXTERNAL] Re: [LabKey/onprcEHRModules] 23.11 fb surgical rounds observation (PR #1108)


@labkey-martyp commented on this pull request.

Let me know when you're ready to merge this in and I'll merge in the associated EHR PR.

Please add the new dataset columns to datasets_metadata.xml

—
Reply to this email directly, view it on GitHub<https://urldefense.com/v3/__https:/github.com/LabKey/onprcEHRModules/pull/1108*pullrequestreview-2512677829__;Iw!!Mi0JBg!NePrGXzu_F2v68HKpQJLT_xAvxWHWP_gV9pIc6YFGrMYEg9oujEKyE8oYuYc6SikWVACYCB2MyqhByvZHKZzGIw$>, or unsubscribe<https://urldefense.com/v3/__https:/github.com/notifications/unsubscribe-auth/ASHU2YDZH7ZGK2HZHZNSEBL2GHDH7AVCNFSM6AAAAABMUYIGFGVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDKMJSGY3TOOBSHE__;!!Mi0JBg!NePrGXzu_F2v68HKpQJLT_xAvxWHWP_gV9pIc6YFGrMYEg9oujEKyE8oYuYc6SikWVACYCB2MyqhByvZz5r0Mq0$>.
You are receiving this because you were mentioned.Message ID: ***@***.***>

labkey-martyp commented:
@Ohsudev Got it, I see you copied it over. Looks good.

Ohsudev committed:
Merge branch 'release23.11-SNAPSHOT' into 23.11_fb_SurgicalRoundsObservation

Ohsudev committed:
Modified Surgical Rounds code.

Ohsudev committed:
Modified Surgical Rounds code.

Ohsudev committed:
Modified Surgical Rounds code.

Ohsudev committed:
Modified Surgical Rounds to include "type" as new query qualifier to differentiate between surgery and clincal entries.

Ohsudev committed:
Modified Surgical Rounds to include "type" as new query qualifier to differentiate between surgery and clincal entries.

Ohsudev committed:
Modified Surgical Rounds to include "type" as new query qualifier to differentiate between surgery and clincal entries.

Ohsudev committed:
Modified Surgical Rounds to include "type" as new query qualifier to differentiate between surgery and clincal entries.

Ohsudev committed:
Modified Compliance Overdue/Soon reports.

Ohsudev committed:
Merge branch 'release23.11-SNAPSHOT' into 23.11_fb_SurgicalRoundsObservation

Ohsudev committed:
Modified Surgical Rounds drop down selection listings.

Ohsudev committed:
Modified Surgical Rounds drop down selection listings.

Ohsudev committed:
Modified Surgical Rounds drop down selection listings.

Ohsudev committed:
Modified Surgical Rounds drop down selection listings.
